charts/kube-master/charts/etcd/templates/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+{{/* vim: set filetype=gotexttmpl: */ -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  openstack-auth-url: {{ required "missing openstack-auth-url" .Values.openstack.authURL | b64enc }}
+  openstack-username: {{ required "missing openstack-username" .Values.openstack.username | b64enc }}
+  openstack-password: {{ required "missing openstack-password" .Values.openstack.password | b64enc }}
+  openstack-domain-name: {{ required "missing openstack-domain-name" .Values.openstack.domainName | b64enc }}
+  openstack-project-id: {{ required "missing openstack-project-id" .Values.openstack.projectID | b64enc }}
```

Comment:
What project do we end up having configured here? Is it our control-plane project or the customers project?

Reply:
Both, customer backups should be on customer quota.